freeipa/values.yaml

```diff
@@ -63,7 +63,7 @@ ingress:
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
   hosts:
-    - host: chart-example.local
+    - host: freeipa.improwised.dev
```

Comment:
This should remain a dummy value. We will override it in our deployment instead.

Reply:
done!